apps/web/components/Organization/Wrapper/OrganizationWrapper.tsx

```diff
@@ -183,16 +183,15 @@ export const getThemeConfig = (
       ? 'organization'
       : 'default'
 
-  if (blueberryThemes.includes(theme ?? ''))
+  if (blueberryThemes.includes(theme ?? '') && !usingDefaultHeader)
```

Comment:
Could we maybe do an early return since in all cases if the default header is chosen then we'd like to use the light mode header variant, right?